tests/dev/03_Population_Pediatric_Raltegavir/test-example-03-pk-parameters.R

```diff
@@ -55,10 +60,22 @@ myPediatricWorkflow$activateTasks(tasks = "plotPKParameters")
 myPediatricWorkflow$inactivateTasks(tasks = c("simulatePopulation", "populationPKParameters"))
 
 # Task "plotPKParameters": has xParameters and yParameters options
-# - xParameters are path of demography parameters for VPC like range plots (use ospsuite::StandardPath)
-# - yParameters are selected PK parameters to be exported in output (use ospsuite::StandardPKParameter)
+# Use <get/set><X/Y>parametersFor to check or update these options
+# Caution: yParameters are actually Output objects
+
+getXParametersForPkParametersPlot(myPediatricWorkflow)
+getYParametersForPkParametersPlot(myPediatricWorkflow)
+
+setXParametersForPkParametersPlot(myPediatricWorkflow, c(StandardPath$Age, StandardPath$Weight))
+
+myOutput <- Output$new(
+  path = "Organism|PeripheralVenousBlood|Raltegravir|Plasma (Peripheral Venous Blood)",
+  displayName = "Venous Blood",
+  pkParameters = c("AUC_inf", "C_max"),
+  pkParametersDisplayName = c("My AUC infinity", "My Cmax"),
+  pkParametersDisplayUnit = c("nmol*min/l", "nmol")

```

Comment:
@pchelle Let's consolidate pk parameters input structure as well. Instead of passing 1 to 3 lists (`pkParameters`, `pkParametersDisplayName`, `pkParametersDisplayUnit `) I would pass a list of structures with this 3 values. Thus e.g. something like this:
```
pk1 = pkParameter$new("AUC_inf", "My AUC infinity", "nmol*min/l")
pk2 = pkParameter$new("C_max", "My Cmax", "nmol")
myOutput <- Output$new(
  path = "Organism|PeripheralVenousBlood|Raltegravir|Plasma (Peripheral Venous Blood)",
  displayName = "Venous Blood",
  pkParameters = c(pk1, pk2)
)
```
- if `pkParameter` is already used in the ospsuite package, we could call it `pkParameterInfo `or something like that
- `displayName `and `displayUnit `in `pkParameter$new()` should be still optional

Reply:
why are displayName and displayUnit used here? Didn't we say it's a global setting?

Reply:
>why are displayName and displayUnit used here? Didn't we say it's a global setting?

Did we? For the _display units_ it makes sense for me **not** to define them globally (or at least to have the possibility to **override** the global setting). User might want to have the same pk parameter in different display units for the different curves (due to different order of magnitudes of calculated values or whatever)

Reply:
Maybe. For me it does not make sense. If you are creating plots that you want to compare and you are changing the UNIT used on the plot, there is no way for you to see how stuff has changed based on your scenario

And do we really need this for a V1?

Reply:
and display name DEFINITELY does not make sense

Reply:
Agree for the display name